scripts/index.js

```diff
@@ -0,0 +1,166 @@
+
+// HELPER FUNCTION //
+function numsChangeColor() {
+    let temp = document.querySelector('#givenTemp');
+    
+    if (state.givenTemp >= 100) {
+        temp.style.color = "black"
+    } else if ((state.givenTemp < 100) && (state.givenTemp >= 70)) {
+        temp.style.color = "red"
+    } else if ((state.givenTemp < 70) && (state.givenTemp >= 60)) {
+        temp.style.color = "orange"
+    } else if ((state.givenTemp < 60) && (state.givenTemp >= 50)) {
+        temp.style.color = "yellow"
+    } else if ((state.givenTemp < 50) && (state.givenTemp >= 40)) {
+        temp.style.color = "green"
+    } else if ((state.givenTemp < 40) && (state.givenTemp >= 30)) {
+        temp.style.color = "blue"
+    } else {
+        temp.style.color = "purple"
+    }
+};
+
+////////////////////////////// WAVE 2 //////////////////////////////////
+////////////////// EVENT HANDLER (UP/DOWN TEMPERATURE)//////////////////
+// SELECT THE EVENT TARGET
+// document.getElementById('givenTemp') || document.querySelector("#givenTemp");
+
+////////////////////////////// WAVE 2 //////////////////////////////////
+/////////////// METHODS (UP/DOWN TEMPERATURE) BEHAVIORS ////////////////
+const state = {
+    givenTemp: 70
+};
+const addTemp = (event) =>  {       // Add temp UP Behavior
+    state.givenTemp += 1;
+    const tempUp = document.querySelector("#givenTemp");
+    tempUp.textContent = state.givenTemp;
+    numsChangeColor();
+}
+const subTemp = (event) => {        // Add temp DOWN Behavior
+    state.givenTemp -= 1;
+    const tempDown = document.querySelector("#givenTemp");
+    tempDown.textContent = state.givenTemp;
+    numsChangeColor();
+};
+
+// REGISTER ALL EVENTS // use the .registerEventHandler
+const registerEventHandlers = (event) => {              // EACH EVENT needs a .addEventListener - listens fot the event i.e. "click" then performs a function
+                                                        // WAVE 2 // reigster the upButton
+    const upButton = document.querySelector("#addUpButton");
+    upButton.addEventListener("click", addTemp);
+                                                        // WAVE 2 // reigster the downButton
+    const downButton = document.querySelector("#addDownButton");
+    downButton.addEventListener("click", subTemp);
+                                                        // WAVE 5 // register the reset / submit button
+    const submitButton = document.querySelector("#submitButton");
+    submitButton.addEventListener("click", switchCity);
```

Comment:
This isn't quite the behavior the requirements were asking for.  For this section we were looking for two different pieces of functionality:
1. An input listener on the text box that updates as the user enters text (https://www.w3schools.com/jsref/event_oninput.asp)
2. a reset button that set the name back to a starting default